go.mod

```diff
@@ -3,24 +3,40 @@ module github.com/gruntwork-io/go-commons
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.38.0 // indirect
```

Comment:
Merging with master had conflicts at `go.mod` and `go.sum`,  running `go mod tidy` resulted in these changes

Reply:
Thanks! Is there any action needed?